docs/DeveloperGuide.md

```diff
@@ -19,6 +19,14 @@ prospective users to plan their exercise routines.
 This technical document is meant for current and future developers of WerkIt! as a reference point on the design,
 implementation, and other technical and non-technical aspects of the application.
 
+When reading this document, there are several icons that you may encounter. 
+Below are the icons and their meanings:
```

Comment:
Perhaps a (H3) subheader for this section may help distinguish it from the introductory notes of the above part better.